internal/pkg/project/ignore/ignore.go

```diff
@@ -0,0 +1,55 @@
+package ignore
+
+import (
+	"strings"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+func (f *File) IgnoreConfigsOrRows() error {
+	return f.applyIgnoredPatterns()
+}
+
+func (f *File) parseIgnoredPatterns() []string {
+	var ignorePatterns []string
+	lines := strings.Split(f.rawStringPattern, "\n")
+	for _, line := range lines {
+		trimmedLine := strings.TrimSpace(line)
+		// Skip empty lines and comments
+		if trimmedLine != "" && !strings.HasPrefix(trimmedLine, "#") {
+			ignorePatterns = append(ignorePatterns, trimmedLine)
+		}
+	}
+
+	return ignorePatterns
+}
+
+// applyIgnorePattern applies a single ignore pattern, marking the appropriate config or row as ignored.
+func (f *File) applyIgnorePattern(ignoreConfig string) error {
+	parts := strings.Split(ignoreConfig, "/")
+
+	switch len(parts) {
+	case 2:
+		// Ignore config by ID and name.
+		configID, componentID := parts[1], parts[0]
+		f.state.IgnoreConfig(configID, componentID)
+	case 3:
+		// Ignore specific config row.
+		configID, rowID := parts[1], parts[2]
+		f.state.IgnoreConfigRow(configID, rowID)
+	default:
+		return errors.Errorf("invalid ignore ignoreConfig format: %s", ignoreConfig)
+	}
+
+	return nil
+}
+
+// applyIgnoredPatterns parses the content for ignore patterns and applies them to configurations or rows.
+func (f *File) applyIgnoredPatterns() error {
+	for _, pattern := range f.parseIgnoredPatterns() {
+		if err := f.applyIgnorePattern(pattern); err != nil {
+			continue
+		}
+	}
+	return nil
+}
```

Comment:
```suggestion
// applyIgnoredPatterns parses the content for ignore patterns and applies them to configurations or rows.
func (f *File) applyIgnoredPatterns() error {
	for _, pattern := range f.parseIgnoredPatterns() {
		if err := f.applyIgnorePattern(pattern); err != nil {
			continue
		}
	}
	return nil
}

func (f *File) parseIgnoredPatterns() []string {
	var ignorePatterns []string
	lines := strings.Split(f.rawStringPattern, "\n")
	for _, line := range lines {
		trimmedLine := strings.TrimSpace(line)
		// Skip empty lines and comments
		if trimmedLine != "" && !strings.HasPrefix(trimmedLine, "#") {
			ignorePatterns = append(ignorePatterns, trimmedLine)
		}
	}

	return ignorePatterns
}

// applyIgnorePattern applies a single ignore pattern, marking the appropriate config or row as ignored.
func (f *File) applyIgnorePattern(ignoreConfig string) error {
	parts := strings.Split(ignoreConfig, "/")

	switch len(parts) {
	case 2:
		// Ignore config by ID and name.
		configID, componentID := parts[1], parts[0]
		f.state.IgnoreConfig(configID, componentID)
	case 3:
		// Ignore specific config row.
		configID, rowID := parts[1], parts[2]
		f.state.IgnoreConfigRow(configID, rowID)
	default:
		return errors.Errorf("invalid ignore ignoreConfig format: %s", ignoreConfig)
	}

	return nil
}
```
TL;DR the applyIgnoredPatterns should be on top

Reply:
[2c14c77](https://github.com/keboola/keboola-as-code/pull/2136/commits/2c14c7783abac54d60b6f12e32664d6b2704cd74)